images/csp_violation_report_service/app/config/application.py

```diff
@@ -0,0 +1,16 @@
+from masonite.environment import env
+
+
+KEY = env("APP_KEY", "-RkDOqXojJIlsF_I8wWiUq_KRZ0PtGWTOZ676u5HtLg=")
```

Comment:
What do you think of the idea of not passing a default APP_KEY here and then failing if one doesn't exist?

The scenario I'm thinking of is us accidentally using this `APP_KEY` live because we forget to specify one in the `.env`.